data.js

```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Articles wrote \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-02-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+ 
+                     "\n- Were lobbying for the Paris Agreement."+
+                     "\n- Helped to implement first binding environmental laws in 16 South American states."+
+                     "\n Read about (more)[https://www.wri.org/our-work/outcomes] stories of success though out the past years.",
```

Comment:
1. `Read more stories`, not `Read about more stories`
2. though out -> you probably mean throughout but "stories from the past would be the correct version. I suggest to leave out, all stories are from the past.

Reply:
done